pkg/arvo/gen/solid.hoon

```diff
@@ -15,7 +15,8 @@
         arg=$@(~ [top=path ~])
         dub=_|
     ==
-:-  %noun
+:-  %pill
+%-  pill:pill
```

Comment:
The reason to make `+solid` produce a well-typed pill is that we now have circumstances where it's reasonable to write it to clay or pass it into aqua.  This lets you write `:aqua +solid` instead of `:aqua &pill +solid`.

I haven't noticed any performance impact, the pill type isn't very specific, so the clam doesn't need to do much.  However, I'm pretty sure I can just change `+file-ovum` to produce a `unix-event`, then it can be a cast.